packages/components/src/components/tablist/tablist.stories.ts

```diff
@@ -0,0 +1,125 @@
+import type { Meta, StoryObj, Args } from '@storybook/web-components';
+import { action } from '@storybook/addon-actions';
+import '.';
+import { html } from 'lit';
+import { disableControls, hideControls } from '../../../config/storybook/utils';
+import { TAB_VARIANTS } from '../tab/tab.constants';
+import '../badge';
+
+const render = (args: Args) => html`
+<mdc-tablist
+  @change="${action('onchange')}"
+  @keyup="${action('onkeyup')}"
+  @keydown="${action('onkeydown')}"
+  active-tabid="${args['active-tabid']}"
+  aria-label="Media types">
+  <mdc-tab
+    variant=${args.tabvariant}
+    text="Photos"
+    icon-name="add-photo-bold"
+    tabid="photos-tab"
```

Comment:
```suggestion
    tab-id="photos-tab"
```
Please update it accordingly across this file.

Reply:
updated